src/orquestra/sdk/_base/_workflow.py

```diff
@@ -158,42 +158,50 @@ def local_run(self) -> _R:
         _dsl.DIRECT_EXECUTION = False
         return result
 
-    def prepare(
+    def run(
         self,
-        config: Union[_api.RuntimeConfig, str],
+        config: Optional[Union[_api.RuntimeConfig, str]] = None,
         project_dir: Optional[Union[str, Path]] = None,
         workspace_id: Optional[WorkspaceId] = None,
         project_id: Optional[ProjectId] = None,
     ) -> _api.WorkflowRun:
         """
-        "Prepares" workflow for running. Call ".start()" on the result to
-        schedule the workflow for execution.
+        Schedules workflow for execution.
 
         Args:
-            config: SDK needs to know where to execute the workflow. The config
-                contains the required details. This can be a RuntimeConfig object, or
-                the name of a saved configuration.
+            config: SDK needs to know where to execute the workflow. This
+                objects contains the required details.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏I'm not sure what this comment change means